v2.1/admin-ui-access-and-navigate.md

```diff
@@ -8,6 +8,53 @@ toc: false
 
 ## Access the Admin UI
 
+By default, CockroachDB allows all users to access and view the Admin UI. For added security, you can choose to enable user authentication so that only authorized users can access and view the Admin UI.
+
+<div class="filters filters-big clearfix">
+  <button class="filter-button" data-scope="with-user-auth"><strong>Enable user authentication</strong></button>
+  <button class="filter-button" data-scope="without-user-auth"><strong>Ignore user authentication</strong></button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="with-user-auth">
+
+User authentication can be enabled only for secure clusters. To enable user authentication:
+
+### Step 1. Start a secure cluster with the user authentication environment variable set for each node:
+
+To start a secure cluster, first [create security certificates](secure-a-cluster.html#step-1-create-security-certificates).
+
+Then set the environment variable, `COCKROACH_EXPERIMENTAL_REQUIRE_WEB_LOGIN=TRUE`, while [starting each node](secure-a-cluster.html#step-2-start-the-first-node) in the cluster:
```

Comment:
I thought we were not going to document this, since we'll be turning it off before 2.1 is released, and just tell our alpha testers about it in the meantime.

Reply:
@vilterp Another argument was to document the state-of-the-product at the time of alpha and editing it as and when we remove the variable. That made more sense from the user's perspective, so went with that option. Are there any downsides to documenting the variable for now?

Reply:
Only downside is that you'll have to do some editing, i.e. removing all references to this environment variable and just saying that "if your cluster is secure, the admin ui will be behind login; otherwise it will be open"

Reply:
That's okay. I don't mind doing that :)